iroha_macro/iroha_derive/tests/from_variant/02-double-from.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0119]: conflicting implementations of trait `std::convert::From<Variant1>` for type `Enum`:
```

Comment:
Are these files must be here?

Reply:
Yeah, it is needed for comparing compiler output for some tests. Forgot to add them before